reactive-banana/src/Reactive/Banana/Prim/Graph.hs

```diff
@@ -18,8 +25,19 @@ import           Data.Maybe
     Graphs and topological sorting
 ------------------------------------------------------------------------------}
 data Graph a = Graph
-    { children :: Map.HashMap a [a]
+    { -- | The mapping from each node to the set of nodes reachable by an out-edge. If a node has no out-edges, it is
```

Comment:
Just a bit of documentation about how this particular flavor of directed graph representation works.